rust-runtime/aws-smithy-http/src/byte_stream.rs

```diff
@@ -628,14 +634,167 @@ mod tests {
             .await
             .expect("should have some data")
             .expect("read should not fail");
-        // The size of one read should be equal to that of the buffer size
+        // The size of one read should be equal to that of the length
```

Comment:
I think `buffer size` was correct?

Reply:
I don't remember why I changed this in the first place lol